### PR TITLE
Restrict building the pkg/fileserver implementation to windows

### PR DESCRIPTION
**Pull request by siretart**
This is to avoid p9 dependencies when not building for windows

#### Does this PR introduce a user-facing change?

no, I've applied this patch to the Debian package to avoid unnecessary dependencies.

### CODE REVIEW

File: pkg/fileserver/server_windows.go [hunk 1]
```diff
@@ -1,3 +1,5 @@
+//go:build windows
+// +build windows
```
Comment (Luap99) [inline pkg/fileserver/server_windows.go:2]: this file is already called _windows.go so it should already be only compiled on windows anyway so this tag should not be necessary

File: pkg/fileserver/plan9/serve.go [hunk 2]
```diff
@@ -1,3 +1,5 @@
+//go:build windows
+// +build windows
```
Comment (Luap99) [inline pkg/fileserver/plan9/serve.go:2]: here as well I would juts rename the file to serve_windows.go

**Comment by siretart** [comment]
good idea, implemented @Luap99 suggestions

**Comment by TomSweeneyRedHat** [comment]
LGTM

Two failing sqlite tests, I can't imagine the failures are related to this change.

**Review by Luap99 — APPROVED** [review]
/lgtm

**Pull request merged**